appscale/tools/agents/azure_agent.py

```diff
@@ -1473,29 +1476,25 @@ def get_params_from_args(self, args):
       self.PARAM_INSTANCE_TYPE: args[self.PARAM_INSTANCE_TYPE],
       self.PARAM_GROUP: args[self.PARAM_GROUP],
       self.PARAM_KEYNAME: args[self.PARAM_KEYNAME],
-      self.PARAM_RESOURCE_GROUP: args[self.PARAM_RESOURCE_GROUP],
-      self.PARAM_STORAGE_ACCOUNT: args[self.PARAM_STORAGE_ACCOUNT],
+      self.PARAM_RESOURCE_GROUP: args.get(self.PARAM_RESOURCE_GROUP,
+                                          self.DEFAULT_RESOURCE_GROUP),
```

Comment:
What is the advantage of having a default resource group as opposed to failing if the resource group is not defined? I would imagine most users do not have access to 'appscalegroup'.

Reply:
I would be fine with failing without a resource group, I was just moving the logic we use below for assigning a default.